internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,60 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	//Создаем поля
+	Steps                 int
+	Duration              time.Duration
+	personaldata.Personal //встроенная структура
+	// Personal из пакета personaldata, у которой есть метод Print().
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	//Разделяем строку на слайс строк
+	sl := strings.Split(datastring, ",")
+	if len(sl) != 2 {
+		return errors.New("slice length is not equal to 2")
+	}
+	// Преобразуем первый элемент в int, второй элемент второй к типу time.Duration
+	st, err := strconv.Atoi(sl[0])
+	if err != nil {
+		return err
```

Comment:
Лучше оборачивать, чтобы потом быстрее находить, где ошибка
```suggestion
		return fmt.Errorf("invalid steps: %w", err)
```